substrate/frame/broker/src/benchmarking.rs

```diff
@@ -939,6 +991,73 @@ mod benches {
 		Ok(())
 	}
 
+	#[benchmark]
+	fn enable_auto_renew() -> Result<(), BenchmarkError> {
+		let _core = setup_and_start_sale::<T>()?;
+
+		advance_to::<T>(2);
+
+		let caller: T::AccountId =
+			T::SovereignAccountOf::maybe_convert(2001).expect("Failed to get sovereign account");
+		T::Currency::set_balance(
+			&caller.clone(),
+			T::Currency::minimum_balance().saturating_add(100u32.into()),
+		);
+
+		let region = Broker::<T>::do_purchase(caller.clone(), 10u32.into())
+			.map_err(|_| BenchmarkError::Weightless)?;
+
+		Broker::<T>::do_assign(region, None, 2001, Final)
+			.map_err(|_| BenchmarkError::Weightless)?;
+		// advance to next bulk sale:
+		advance_to::<T>(6);
+
+		// The most 'intensive' path is when we renew the core upon enabling auto-renewal.
```

Comment:
With max auto renewals. You've done it for rotate_sale, but not for enable/disable autorenew extrinsics

Reply:
Otherwise, the PR looks complete. nice